packages/react-core/src/async-loader.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export interface Container extends Window {
+  init: (module: any) => void;
+}
+
+declare function __webpack_init_sharing__(scope: string): void;
+declare let __webpack_share_scopes__: any;
+
+const ErrorComponent: React.ComponentType<any> = () => {
+  return <span>Error while loading component!</span>;
+};
+
+export function loadComponent(scope: string, module: string) {
```

Comment:
We should make the `ErrorComponent` configurable and use the current `ErrorComponent` implementation as a fallback.